packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Updates minimum Flutter version to 2.10.
+* Updates `win32` to 3.0.0.
```

Comment:
No changes are being made to this package, so there should not be a CHANGELOG change to it.